content/en/security/cspm/findings.md

```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
 
 {{< img src="security/cspm/findings/passed-rules.png" alt="Group and aggregate by resource in search" style="width:100%;">}}
 
+## Mute findings
+
+{{< callout url="" btn_hidden="true" >}}
+  Muted findings is a beta feature available to all CSPM customers. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.
+{{< /callout >}} 
+
+There may be times when a finding does not match the use case for your business, or you choose to accept it as a known risk. To ignore these findings, you can mute the finding for the impacted resource. This allows you to focus on high-severity and critical findings that require your attention.
+
+For example, the ['Block Public Access' feature is enabled for S3 bucket][4] rule evaluates whether a S3 bucket is publicly accessible. If you have a S3 bucket with static assets that are meant to be publicly shared, you can mute the finding for the S3 bucket.
+
+You can mute pass/fail findings at any time. Muted findings do not impact your posture score.
+
+{{< img src="security/cspm/findings/muted-findings.png" alt="The Mute findings dialog box contains fields for specifying the reason and duration of the mute" style="width:100%;">}}
+
+1. On the [finding side panel](#explore-your-cloud-misconfigurations-with-findings), select a resource.
+2. Click **Mute**.
+3. Select a reason for the mute:
+    - Pending fix
+    - False positive
+    - Accepted risk
+    - Other
+4. Enter an optional **Description**.
+5. Select the duration of the mute:
+    - 2 weeks
+    - 1 month
+    - 3 months
+    - Indefinite
+    - Custom
```

Comment:
By listing these here, you're regurgitating what's already on the UI, and committing to keeping the list up to date. What if you said "Select a reason for the mute, for example, a fix is pending, it's a false positive, or it's an accepted risk." and for the duration, don't list them at all.

Reply:
Good point, and I like your suggestion. I'll update the PR with your recommendations.